lib/index.ts

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env -S node -r "ts-node/register"
```

Comment:
Just curious ts-node vs. compiling, which approach is better? I appreciate that ts-node simplifies the development workflow, but I'm uncertain about the trade-offs.

Reply:
Ok yes, that makes more sense. I'll do that.

Reply:
See 3c32160e5a7738f82b4a550bf4d391b70402d094 and 5903497600fd3f0c07ff1e7a1851b3aeee22502f. Specifically for the CI in 5903497600fd3f0c07ff1e7a1851b3aeee22502f I'm not sure if that is what you meant, please let me know. Thanks.

Reply:
All GLTM.